examples/cc_binary/forward_allocator.c

```diff
@@ -0,0 +1,46 @@
+#include <stdint.h>
```

Comment:
should we move this file to a shared location where consumers of the rules can use it as well?

Reply:
Do you want to maintain it as a supported library here, or ask each project that uses it to maintain a version compatible with their rust toolchains? As you can see from the change I made yesterday, it does require updates for new Rust toolchains sometimes, because this is no stable interface to this part of the Rust ABI.

Reply:
Yea I can't say for sure since I'm not a maintainer on this, but as a user I would love it to be maintained here for everyone as opposed to me having to copy paste 🙃

Reply:
I think this is good as an example, but at least right now IMO it doesn't seem like something to add to a shared location and to commit rules_rust to maintain a specific version of this. My main concern is that this is subtle enough that in some non-trivial environments it would require some workspace-specific tweaks that will be hard to maintain and not be universal for all users. For example, we've got an environment where we have to weaken some of the `__rust*` symbols, otherwise we may run into duplicate symbol errors in some configurations when the dependency graph is sufficiently complicated.
I think it's worthwhile to follow-up the conversation about this over at https://github.com/bazelbuild/rules_rust/issues/1238.

@hlopko also may have thoughts about this.